proof/refine/ARM/Finalise_R.thy

```diff
@@ -2705,7 +2705,7 @@ lemma schedContextUnbindTCB_invs'[wp]:
   unfolding schedContextUnbindTCB_def
   apply (rule schedContextUnbindTCB_invs'_helper[simplified] hoare_seq_ext | clarsimp)+
         apply (wp tcbReleaseRemove_invs' tcbReleaseRemove_not_queued
-                  tcbSchedDequeue_nonq tcbSchedDequeue_invs' hoare_vcg_all_lift)+
+                  tcbSchedDequeue_nonq tcbSchedDequeue_invs' hoare_vcg_all_lift | simp)+
```

Comment:
Out of curiosity, any idea what goes wrong if this is `wpsimp`?

Reply:
Probably nothing, will have a look and update it if it works. (I'm just in a habit of making minimal changes because I find it hard to draw the line between removing sorries and doing cleanup. In this case, it's a very small improvement to make and so I should do it).